_data/heroku-landing-page-tabs-images.yml

```diff
@@ -0,0 +1,9 @@
+- id: logs
+  title: Logs
+  content: /assets/img/landing-pages-content/logs.gif
+- id: metrics
+  title: Metrics
+  content: /assets/img/landing-pages-content/metrics.gif
+- id: alarms
+  title: Alarms
+  content: /assets/img/landing-pages-content/alarms.gif
```

Comment:
One of the gifs actually shows our account ID for Phx account. I wonder if this is not desired?

Reply:
Not a big deal